src/features/events/components/EventPopper/SingleEvent.tsx

```diff
@@ -83,7 +87,7 @@ const SingleEvent: FC<SingleEventProps> = ({ event, onClickAway }) => {
       onSelect: () =>
         showConfirmDialog({
           onSubmit: () => {
-            model.deleteEvent();
+            deleteEvent(event.id);
```

Comment:
I don't think it should be necessary to pass the ID here, as it should already be passed to `useEventMutations()`.

This is related to several other comments I've made, but I'm adding multiple in order to try and capture and highlight the full pattern.